emperor/template.py

```diff
@@ -0,0 +1,260 @@
+TEMPLATE_HTML = """
```

Comment:
Great, thanks!


On (Mar-21-16|10:41), Jamie Morton wrote:
> > @@ -0,0 +1,260 @@
> > +TEMPLATE_HTML = """
> 
> :+1: I'll remove it in the next commit, once I get a prototype working.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/451/files/ff29fd2a2d351149a018d47d33341ebc9c9429a6#r56864027